src/main/java/org/gradle/profiler/GradleScenarioInvoker.java

```diff
@@ -30,17 +30,19 @@ public GradleScenarioInvoker(DaemonControl daemonControl, PidInstrumentation pid
     @Override
     public List<Sample<? super GradleBuildInvocationResult>> samplesFor(InvocationSettings settings, GradleScenarioDefinition scenario) {
         ImmutableList.Builder<Sample<? super GradleBuildInvocationResult>> builder = ImmutableList.builder();
-        builder.add(BuildInvocationResult.EXECUTION_TIME);
+        if (scenario.isAndroidStudioSync()) {
+            builder.add(GradleBuildInvocationResult.TOTAL_EXECUTION_TIME);
+            builder.add(GradleBuildInvocationResult.GRADLE_TOOLING_AGENT_EXECUTION_TIME);
+            builder.add(GradleBuildInvocationResult.STUDIO_EXECUTION_TIME);
+        } else {
+            builder.add(BuildInvocationResult.EXECUTION_TIME);
```

Comment:
Why do we now have `EXECUTION_TIME` and `TOTAL_EXECUTION_TIME` as well? I think we should only have `TOTAL_EXECUTION_TIME` now.

Reply:
Changin execution time also affects others. I remove `TOTAL_EXECUTION_TIME`, 🤞  that we don't break any automation that reads csv.